projects/telemetry/inc/nmea.h

```diff
@@ -0,0 +1,76 @@
+#pragma once
+
+// This module is used to parse and represent NMEA data. Used in the GPS module
+
+#include <stdint.h>
+#include <stdio.h>
+#include "status.h"
+
+// This enum contains the list of NMEA sentences (not all of then are supported yet)
+// because they do not contain useful information
+typedef enum {
+  NMEA_UNKNOWN = 0,
+  NMEA_GGA,
+  NMEA_GLL,
+  NMEA_GSA,
+  NMEA_GSV,
+  NMEA_RMC,
+  NMEA_VTG,
+  NMEA_NUM_MESSAGE_IDS
+} NMEA_MESSAGE_ID;
```

Comment:
Please follow our naming standards. This holds for the rest of your code.

Reply:
Thanks, got it

Reply:
ping